RealmWeaver/ModuleWeaver.cs

```diff
@@ -311,7 +309,13 @@ private void WeaveType(TypeDefinition type, Dictionary<string, Tuple<MethodRefer
             return false;
         }
 
-        if (!prop.IsAutomatic())
+        if (prop.IsAutomatic()) 
+        {
+            var objectIdMsg = isObjectId ? "[ObjectId]" : "";
+            var indexedMsg = isIndexed ? "[Indexed]" : "";
+            LogDebug($"Woven {type.Name}.{prop.Name} as a {prop.PropertyType.FullName} {objectIdMsg} {indexedMsg}");
```

Comment:
Shouldn't this log statement be at the bottom of `WeaveProperty`? There are conditions below that might cause the property not to be woven, like `IList` with setter, etc..